src/agents/customer_group.jl

```diff
@@ -610,6 +612,7 @@ function CustomerGroup(input_filename::AbstractString, model_data::HEMData; id =
         initialize_param("total_der_capacity_my_delay_update",model_data.index_y, model_data.index_z, model_data.index_h, index_m),
         initialize_param("total_pv_only_capacity_my_delay_update",model_data.index_y, model_data.index_z, model_data.index_h, index_m),
         initialize_param("total_pv_stor_capacity_my_delay_update",model_data.index_y, model_data.index_z, model_data.index_h, index_m),
+        ParamScalar("B1GM", 100),
```

Comment:
```suggestion
        # TODO: Calculate BIGM from input data
        ParamScalar("B1GM", 100),
```

Reply:
Also, it appears that the new parameter is `B1GM` when I was expecting `BIGM`? (I.e., the digit 1 instead of capital i?)